src/middlewared/middlewared/plugins/network.py

```diff
@@ -2082,6 +2085,13 @@ def ip_in_use(self, choices=None):
         if not choices['loopback']:
             ignore_nics.append('lo')
         ignore_nics = tuple(ignore_nics)
+        static_ips = {}
+        if choices['static']:
+            for i in self.middleware.call_sync('interface.query'):
+                for alias in i.get('failover_virtual_aliases') or []:
```

Comment:
For HA we should only show virtual aliases and not regular aliases. We dont want people binding to each node's IP.